action.yaml

```diff
@@ -32,6 +36,14 @@ runs:
         curl -O -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh && \
         sudo sh install.sh -b /usr/local/bin
 
+    - name: Install grype
+      if: ${{ inputs.generate_vulnerability_report == 'true' }}
+      shell: bash
+      run: |
+        export DEBIAN_FRONTEND=noninteractive && \
+        curl -O -sSfL https://raw.githubusercontent.com/anchore/grype/main/install.sh && \
```

Comment:
(Not necessarily linked to this PR but it seems we use the "main" branch, are we sure each and every commit pushed on this main branch will not cause issue ? I think we should pin the version here to a specific tag)

Reply:
We talk about it yesterday, but @igorscality  prefer to always use latest.